app/transformers/api/v3/transformer.rb

```diff
@@ -1,7 +1,19 @@
 class Api::V3::Transformer
   class << self
+    def redirect_to_deduped_patient(attributes)
+      # NOTE: Move this to a different layer if/when this becomes more complex
+      deduped_record = DeduplicationLog.find_by_deleted_record_id(attributes["id"])&.deduped_record
```

Comment:
I think it's encouraged to move away from the magic `find_by_something` methods, to the less-meta-programmy `find_by(attribute: value)` format. Shall we do that?